KinveyResearchKit/TaskResult.swift

```diff
@@ -29,7 +29,7 @@ open class TaskResult: CollectionResult {
     override open func propertyMapping(_ map: Map) {
         super.propertyMapping(map)
         
-        taskRunUUID <- map[PersistableIdKey]
+        taskRunUUID <- map["taskRunUUID"]
```

Comment:
As I understand, we don't need to enforce. The developer could enforce it themselves if there's a strong need to do so.